### ISSUE TITLE
Limited accessibility / No ARIA instructions

### ISSUE THREAD

**Issue by murb** [dangrossman/bootstrap-daterangepicker#833]
The lack of ARIA instructions reduce the accessibility of this extension. I'm not an expert on the matter but there is good information on MDN: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA Are there any plans on improving ARIA support?

**Comment by dangrossman** [comment]
Not at this time. Thanks for the link.

**Comment by Dakshank** [comment]
+1

**Comment by digitalbart** [comment]
+1
